### ISSUE TITLE
Expected propagation of "copy" of RegEx rules in Autogeneration not met

### ISSUE THREAD

**Issue by scottgit** [DraqueT/PolyGlot#682]
This happens specifically in the Push To Dimension feature. Haven't been able to oreproduce yet. Keep experimenting.

ORIGINAL USER TEXT:
I am experimenting with the right-click and copy rules to other conjugations with the same X (whatever the tense, mood, etc. property it applies to). I have seen two issues:

**First**
When in my conlang font (which is not very conducive to seeing what the regex metacharacters are), I discovered it was not doing an exact copy as I would expect. So my case example was this:

1. The rule in the autogen was left at the .* default for matching
2. The regex of the transformation was as if it were a character that in my conlang, in Latin equivalent, was a forward slash: /
3. The regex, when copied down to other conjugations, ended up as this (brackets added): [/]

Now I realize in regex, those are _equivalent_. But when I see funky characters (i.e. from my conlang equivalent to the left and right bracket characters) in my _copy_ that were not what I put in my _original_, I think something didn't work _right_. It wasn't until I switched the font to a normal type of font that I realized what was happening.

So in short, what is "copied" should be exactly what the user put in the original field to copy. No more, nor less. This is what the user expects.

**Second**
RegEx transformations do not always copy, even though the "rule" copied. That is, I had a rule I tried to propagate ("Add Plural Mark"), and the "name" of the rule did propagate along with the RegEx selection for the rule, but it did not copy the _transformation_ for that rule (transformation area was blank).

Now it may or may not be significant that the transformation rule was similar to another rule that had previously copied (that is, the RegEx for the transformation part was looking for the same character in both rules). In this case, the rules would never have overlapped (though I will have some that do).

**Comment by DraqueT** [comment]
Heyo! As a heads up, this should be corrected now. You can always snag the latest builds at the link below if you would like to give the beta a try!

https://drive.google.com/uc?export=download&id=1o9Htb31I8Flfgh8zNMKe7X4dEoE9YZuL
